### PR TITLE
Add support for dokumentflyt

### PR DESCRIPTION
**Pull request by ivaylomitrev**

### CODE REVIEW

File: src/02 - Object types.md [hunk 1]
```diff
@@ -454,6 +455,26 @@ The following notation is used in the tables below:
 |      |        |                        |          |             |           |  |      |      |       |      |               |                                                                                                                                                                        |           |
 |      | ref    | refRegistrering        |    x     |             |           |  |      |  x   |   x   |      |       x       | <p>&bull; references **Basisregistrering** or **Journalpost**</p><p>&bull; may be partially supported by systems</p><p>&bull; *unlink* not supported in Documaster</p> | &nbsp;    |
 
+### Dokumentflyt
```
Comment (teodoradobreva) [inline src/02 - Object types.md]: Fields `flytFra` and `flytFraBrukerIdent` are missing.

File: src/02 - Object types.md [hunk 2]
```diff
@@ -454,6 +455,26 @@ The following notation is used in the tables below:
...
+| M601 | string | opprettetAv            |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                    |           |
+| M601 | string | opprettetAvBrukerIdent |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                    |           |
+| M660 | string | flytTil                |    x     |             |           |  |  x   |  x   |   x   |  x   |               |                                                    |           |
+| M660 | string | flytTilBrukerIdent     |    x     |             |           |  |  x   |  x   |   x   |  x   |               |                                                    |           |
+| M661 | date   | flytMottattDato        |    x     |             |           |  |  x   |  x   |   x   |  x   |               |                                                    |           |
+| M662 | date   | flytSendtDato          |    x     |      x      |           |  |  x   |  x   |   x   |  x   |               |                                                    |           |
```
Comment (teodoradobreva) [inline src/02 - Object types.md]: Field `flytSendtDato` does not have a default value.

File: src/02 - Object types.md (hunk 1 above)
Comment (ivaylomitrev) [reply src/02 - Object types.md]: Fixed. We had done internal model changes and I had not updated it.

File: src/02 - Object types.md (hunk 2 above)
Comment (ivaylomitrev) [reply src/02 - Object types.md]: Fixed. We had done internal model changes and I had not updated it.

**Comment by ivaylomitrev** [comment]
Squashed and rebased in preparation for merge.

**Pull request merged**